erdpy_network/api_network_provider.py

```diff
@@ -120,6 +120,17 @@ def get_transaction(self, tx_hash: str) -> TransactionOnNetwork:
         transaction = TransactionOnNetwork.from_api_http_response(tx_hash, response)
 
         return transaction
+    
+    def get_transactions(self, address: IAddress, pagination: Optional[IPagination] = None) -> List[TransactionOnNetwork]:
+        default_pagination = DefaultPagination()
```

Comment:
```
pagination = pagination or DefaultPagination()
```

Would this work / simplify the code?

Reply:
This works, but I've simplified the code.